### PR TITLE
Fix lint errors in zkwrapper and zookeeper

### PR DESCRIPTION
**Pull request by ananttibrewal**
Fixed linter issues in zkwrapper, added zkwrapper to the linter's source directories.

### CODE REVIEW

File: zkwrapper/include/zk_lock.h [hunk 1]
```diff
@@ -1,18 +1,18 @@
-//
+// Copyright 2017 Rice University, COMP 413 2017
 // Created by Nicholas Kwon on 10/15/16.
```
Comment (LINKIWI) [inline zkwrapper/include/zk_lock.h]: Please delete these lines

(Authorship can be determined from blame)

File: zkwrapper/source/zk_lock.cc [hunk 2]
```diff
@@ -1,9 +1,9 @@
-//
+// Copyright 2017 Rice University, COMP 413 2017
 // Created by Nicholas Kwon on 10/15/16.
```
Comment (LINKIWI) [inline zkwrapper/source/zk_lock.cc]: ditto here

File: zkwrapper/source/zkwrapper.cc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-//
+// Copyright 2017 Rice University, COMP 413 2017
 // Created by Prudhvi Boyapalli on 10/3/16.
```
Comment (LINKIWI) [inline zkwrapper/source/zkwrapper.cc]: ditto here

**Review by LINKIWI — COMMENTED** [review]
Looks good - some small nits but this looks ready for merge

**Review by ghost — DISMISSED** [review]

**Comment by LINKIWI** [comment]
Looks like this is breaking `DeleteTest`, which was passing as of fff3b4c23dec919379b6dbf8b2d9bb334b385622: https://travis-ci.org/comp413-2017/RDFS/builds/284338306

**Comment by ananttibrewal** [comment]
Fixed lint errors in zookeeper directory.
Added zookeeper to list of directories for linter to check.

**Review by LINKIWI — APPROVED** [review]
Looks reasonable

In the future please update by rebasing instead of merging. When you create a merge commit it makes the diffset unnecessarily large and makes it a lot more difficult to blame changes within a single PR

**Pull request merged**